README.md

```diff
@@ -17,4 +17,108 @@ These expectations are as follows:
+This can be accomplished by modifying the rows of the plan like so:
+
+```julia
+edf = EDF.File(my_edf_file_path)
+plans = plan_edf_to_onda_samples(edf; label=my_labels)
+
+function fix_millivolts(plan)
+    if plan.sample_unit == "millivolt" && plan.kind == "eeg"
+        sample_resolution_in_unit = plan.sample_resolution_in_unit * 1000
+        sample_offset_in_unit = plan.sample_offset_in_unit * 1000
+        return Tables.rowmerge(plan; sample_unit="microvolt",
+                               sample_resolution_in_unit,
+                               sample_offset_in_unit)
+    else
+        return plan
+    end
+end
+
+new_plan = map(fix_millivolts, Tables.rows(plans))
+samples, plan_executed = edf_to_onda_samples(edf, new_plan)
+```
+
+As another, similar example, sometimes EMG channels get recorded with different physical units.
+In such a case, OndaEDF will store them with different `kind` values (`emg_1`, `emg_2`, etc.).
+This can be corrected in a similar way, for exmaple by converting millivolts to microvolts (adjusting of course depending on the nature of your dataset) and re-grouping into Onda signals:
+
+```julia
+edf = EDF.File(my_edf_file_path)
+plans = plan_edf_to_onda_samples(edf; label=my_labels)
+
+function fix_emg(plan)
+    if startswith(plan.kind, "emg")
+        if plan.sample_unit == "millivolt"
+            sample_resolution_in_unit = plan.sample_resolution_in_unit * 1000
+            sample_offset_in_unit = plan.sample_offset_in_unit * 1000
+            plan = Tables.rowmerge(plan; sample_unit="microvolt",
+                                   sample_resolution_in_unit,
+                                   sample_offset_in_unit)
+        end
+        return Tables.rowmerge(plan; kind="emg")
+    else
+        return plan
+    end
+end
+
+new_plan = map(fix_emg, Tables.rows(plans))
+# re-compute the grouping of EDF signals into Onda signals:
+new_plan = OndaEDF.plan_edf_to_onda_signal_groups(new_plan)
+samples, plan_executed = edf_to_onda_samples(edf, new_plan)
+```
+
+### Pre-processing signal headers
+
+Sometimes non-standard usage of the label and transducer type fields makes automatic matching difficult.
+In such cases, you can _preprocess_ the signal headers before generating a plan.
+For example, in a situation where the transducer type and labels are switched, you can switch them back before planning:
+
+```julia
+edf = EDF.File(my_edf_file_path)
+
+function corrected_header(signal::EDF.Signal)
+    header = signal.header
+    return Tables.rowmerge(header; 
+                           label=header.transducer_type, 
+                           transducer_type=header.label)
+end
+
+plans = map(plan_edf_to_onda_samples ∘ corrected_header, edf.signals)
+grouped_plans = OndaEDF.plan_edf_to_onda_signal_groups(plans)
+samples, plan_executed = edf_to_onda_samples(edf, grouped_plans)
+```
```

Comment:
this scenario shows how a user can preprocess the signal headers before planning